front-end/gitaly/server/server.go

```diff
@@ -40,6 +40,7 @@ import (
 	"google.golang.org/grpc/reflection"
 )
 
+// Server interface
```

Comment:
@stepanvalyavskiy this comment doesn't explain anything, it just repeats type name)